app/src/main/java/com/sopt/geonppang/di/KakaoModule.kt

```diff
@@ -0,0 +1,38 @@
+package com.sopt.geonppang.di
```

Comment:
모듈을 따로 만든 이유는?! 그리고 이미 authInterceptor가 있긴 합니다! 그리고 제 생각에 이 부분은 인터셉팅을 할 필요가 없어보여요. @Header로 구현 불가능한가요?

Reply:
음 어노테이션을 정확하게 쓰는 방법을 더 공부해보겠습니다.

```suggestion
package com.sopt.geonppang.difun providesInterceptor(): Interceptor =
        Interceptor { chain ->
            with(chain) {
                proceed(
                    request()
                        .newBuilder()
                        .addHeader(
                            "accesstoken",
                            dummyAccessToken
                        )
                        .addHeader("Content-Type", "Application/json")
                        .build()
                )
            }
        }
```
이런식으로 쓰는 걸 보고 이게 addHeader() 헤더에 넣는 적절한 방법이라고 생각했는데 어렵군여 😢

Reply:
```suggestion
package com.sopt.geonppang.di
override fun intercept(chain: Interceptor.Chain): Response {
        val originalRequest = chain.request()
        val authRequest =
            originalRequest.newBuilder().addHeader("Authorization", BuildConfig.ACCESS_TOKEN)
                .build()
        val response = chain.proceed(authRequest)

        when (response.code) {
            401 -> {
                // TODO 토큰 재발급 api 연동
            }
        }
        return response
    }
```
다른 api 는 여기서 액세스 토큰 넣어주는 것 같긴 합니당